### PR TITLE
My ai

### PR DESCRIPTION
**Pull request by aloncarmel**
- Adding open ai imge url
- Adding open ai imge url
- ai

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### CODE REVIEW

**Comment by vanessa-cooper** [comment]
On it! I'll review the changes as soon as possible!

**Review by vanessa-cooper — CHANGES_REQUESTED** [review]
Looks like we have a few checks failing. Please try again and make sure they all pass.

**Comment by aloncarmel** [comment]
בחייט נס. תאשרי, אני פה עייף ורוצה לישון

**Comment by aloncarmel** [comment]
נס תעזרי לי נננסנסססססססס איפה הלכת לישון כבר

**Comment by aloncarmel** [comment]
ערה?

**Comment by aloncarmel** [comment]
נס תעני לי נס

File: backend/package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "passport-local": "1.0.0",
     "request": "2.69.0",
     "slug": "0.9.1",
-    "underscore": "1.8.3"
+    "underscore": "1.8.3",
+    "openai":"*"
```
Comment (or-yam) [inline backend/package.json:37]: Why not to specify the version ?

File: backend/routes/api/items.js [hunk 2]
```diff
@@ -4,8 +4,16 @@ var Item = mongoose.model("Item");
...
+
 const { sendEvent } = require("../../lib/event");
 
+const { Configuration, OpenAIApi } = require("openai");
+const configuration = new Configuration({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+const openai = new OpenAIApi(configuration);
+
+//Done j
```
Comment (or-yam) [inline backend/routes/api/items.js:16]: 🤨

**Review by or-yam — APPROVED** [review]
LGTM 🤖

Alt Text

File: backend/routes/api/items.js (hunk 2 above)
Comment (aloncarmel) [reply backend/routes/api/items.js]: Fixed

File: backend/package.json (hunk 1 above)
Comment (aloncarmel) [reply backend/package.json]: Fixed

**Review by or-yam — APPROVED** [review]

**Review by vanessa-cooper — APPROVED** [review]
Looking good! You can merge the PR now.

**Pull request merged**